agr_literature_service/lit_processing/oneoff_scripts/make_subset_db/make_subset.py

```diff
@@ -14,24 +14,102 @@
     CopyrightLicenseModel
 
 )
+test_reference_curies = [
+    'AGRKB:101000000656561',
+    'AGRKB:101000000650977',
+    'AGRKB:101000000661443',
+    'AGRKB:101000000679676',
+    'AGRKB:101000000654341',
+    'AGRKB:101000000589922',
+    'AGRKB:101000000863609',
+    'AGRKB:101000000946299',
+    'AGRKB:101000000594062',
+    'AGRKB:101000000872479',
+    'AGRKB:101000000865324']
+
 ALL_OUTPUT = 2  # For verbosity
 helptext = r"""
     env PSQL_XXX used for Target subset db details.
     env ORIG_XXX used for source reference db details.
     If ORIG_XXX is defined use that for source db details.
-    If ORIG_XXX NOT defined then use PSQL_XXX details.
+    If ORIG_XXX NOT defined then PSQL_XXX details wil be used.
     In most cases ORIG_XXX will not be needed but this is incase
     the target and source databases are not on the same server.
 
 """
 parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter, description=helptext)
 parser.add_argument('-l', '--limit', help='Limit to x references', type=int, default=2000, required=False)
 parser.add_argument('-v', '--verbose', help="verbosity, minumum=0, headlines=1, full=2,", type=int, default=1, required=False)
+parser.add_argument('-d', '--dump_dir', help='Directory to dump schemas and dbs too', type=str, required=True)
```

Comment:
probably typo "and dbs to", but maybe not

Reply:
fixed. Thanks